newton/_src/sensors/warp_raytrace/render.py

```
@@ -20,37 +20,45 @@
 if TYPE_CHECKING:
     from .render_context import ClearData, RenderContext
 
 
 @wp.func
-def tid_to_tile_coord(tid: wp.int32, num_worlds: wp.int32, num_cameras: wp.int32, tile_size: wp.int32, width: wp.int32):
-    num_views_per_pixel = num_worlds * num_cameras
-    num_pixels_per_tile = tile_size * tile_size
-    num_tiles_per_row = width // tile_size
+def tid_to_coord_tiled(
+    tid: wp.int32,
+    num_worlds: wp.int32,
```

Ruff (0.14.13)

Unused function argument: `num_worlds`

(ARG001)